services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/Extensions.kt

```diff
@@ -16,3 +17,14 @@ fun <T> String.asJsonResource(type: Class<T>): T =
 fun Float.roundToOneDecimal(): Float {
     return BigDecimal(this.toDouble()).setScale(1, RoundingMode.HALF_UP).toFloat()
 }
+
+internal fun ConstraintViolationException.toHankeError(default: HankeError): HankeError {
+    val violation = constraintViolations.firstOrNull { constraintViolation ->
+        constraintViolation.message.matches(HankeError.CODE_PATTERN)
+    }
+    return if (violation != null) {
+        HankeError.valueOf(violation)
+    } else {
+        HankeError.HAI1011
```

Comment:
HankeController had 1002 here? Does the HankeController-case not end up in this function (and only geometry side ends here)?

Reply:
Right! That "else" should return "default" (given HankeError parameter)